lib/core/src/Cardano/Wallet.hs

```diff
@@ -2941,3 +2942,7 @@ instance HasSeverityAnnotation TxSubmitLog where
         MsgRetryPostTxResult _ (Right _) -> Info
         MsgRetryPostTxResult _ (Left _) -> Debug
         MsgProcessPendingPool msg -> getSeverityAnnotation msg
+
+-- | Empty fluff.
+nullFluff :: Monoid a => a
+nullFluff = mempty
```

Comment:
Hi @rvl 

It seems this doesn't cause weeder to fail, at least not in [this build](https://buildkite.com/input-output-hk/cardano-wallet/builds/16364#24482922-1720-469b-9589-5a4f096483df).

Perhaps I've misunderstood something?

Reply:
Yes - https://github.com/input-output-hk/cardano-wallet/pull/2851#issuecomment-906917880